# Represents the status of the entire system.

# Running processes
Process[] processes

# Total GPU usage percentage, if available.
float32 gpu_usage

# CPU temperature, in C
float32 cpu_temp
# GPU temperature, in C
float32 gpu_temp
